src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/iOS/MediaElementRenderer.ios.cs

```diff
@@ -366,39 +358,29 @@ protected override void OnElementChanged(ElementChangedEventArgs<MediaElement> e
 
 			if (e.OldElement != null)
 			{
-				e.OldElement.PropertyChanged -= OnElementPropertyChanged;
-				e.OldElement.SeekRequested -= MediaElementSeekRequested;
-				e.OldElement.StateRequested -= MediaElementStateRequested;
-				e.OldElement.PositionRequested -= MediaElementPositionRequested;
+				Element.PropertyChanged -= OnElementPropertyChanged;
+				Element.SeekRequested -= MediaElementSeekRequested;
+				Element.StateRequested -= MediaElementStateRequested;
+				Element.PositionRequested -= MediaElementPositionRequested;
```

Comment:
I guess we still should use **e.OldElement** otherwise we will get NullReferenceException when Renderer unassigned from Element.

Reply:
Done